Mathlib/AlgebraicTopology/FundamentalGroupoid/FundamentalGroup.lean

```diff
@@ -85,4 +85,8 @@ abbrev fromPath {X : TopCat} {x : X} (p : Path.Homotopic.Quotient x x) : Fundame
   fromArrow p
 #align fundamental_group.from_path FundamentalGroup.fromPath
 
+/-- The homomorphism between fundamental groups induced by a continuous map. -/
+def map (f : C(X, Y)) : FundamentalGroup X x₀ →* FundamentalGroup Y (f x₀) :=
+  (FundamentalGroupoid.map f).mapAut x₀
```

Comment:
This probably needs a `y₀ = f x₀` version to be pleasant enough to work with, but let it be for now.